android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -2153,6 +2162,12 @@ public void setDrmLicenseHeader(String[] header){
 
     public void setAnalyticsMeta(ReadableMap analyticsData) {
         this.analyticsMeta = analyticsData;
+
+        if (analyticsData != null && analyticsData.getBoolean("contentIsLive") && srcChanged) {
```

Comment:
Idea is to manage live tv youbora with old logic....changes made to make youbora work with VOD doesnt quite support LIVE TV as the player seems to be loading twice because of assetInfo prop in react native side.